packages-exp/auth-exp/src/platform_browser/popup_redirect.ts

```diff
@@ -19,14 +19,14 @@ import { SDK_VERSION } from '@firebase/app-exp';
 import * as externs from '@firebase/auth-types-exp';
 import { isEmpty, querystring } from '@firebase/util';
 
+import { _getProjectConfig } from '../api/project_config/get_project_config';
```

Comment:
unused import, run your linter

Reply:
Linter not picking it up for some reason